data/words/strange_words.json

```diff
@@ -0,0 +1,65 @@
+{
+    "description": "Do you know the feeling when you repeat some word many times and it starts to sound weird? Below is the list of some of the strangest sounding words that people submitted during my Intro to Computational Media Class at ITP, NYU.",  
```

Comment:
Might be worth providing this Wikipedia URL for _semantic satiation_ in the description: https://en.wikipedia.org/wiki/Semantic_satiation